hack/windows/Helper.psm1

```diff
@@ -142,6 +142,12 @@ function Install-AntreaAgent {
     kubectl config --kubeconfig=$AntreaEtc\antrea-agent.antrea.kubeconfig set-credentials antrea-agent --token=$TOKEN
     kubectl config --kubeconfig=$AntreaEtc\antrea-agent.antrea.kubeconfig set-context antrea-agent@antrea --cluster=antrea --user=antrea-agent
     kubectl config --kubeconfig=$AntreaEtc\antrea-agent.antrea.kubeconfig use-context antrea-agent@antrea
+
+    $KUBE_DNS_SERVICE_HOST=$(kubectl get service -n kube-system kube-dns -ojsonpath='{.spec.clusterIP}')
+    $KUBE_DNS_SERVICE_PORT=$(kubectl get service -n kube-system kube-dns -ojsonpath='{.spec.ports[0].port}')
```

Comment:
is `-ojsonpath` valid? you use `-o jsonpath` above

Reply:
I tested on Windows VM and `-ojsonpath` did work. But let's make it better.

Reply:
so now it's `-o=jsonpath=`?

Reply:
I followed the example here: https://kubernetes.io/docs/reference/kubectl/jsonpath/

Reply:
could we just harmonize and use the same everywhere in the script? I don't really care whether it's `-o jsonpath=` or `-o=jsonpath=` as long as it works.

Reply:
Updated. And other `-o=jsonpath` usage in the repo.